src/app/signup/duck/operations.js

```diff
@@ -1,9 +1,11 @@
 import axios from 'axios';
 import { toast } from 'react-toastify';
 import actions from './actions';
+import { actions as loginActions } from '../../login/duck';
 import constants from './constants';
 
 const { setSignupState, setSignupError } = actions;
+const { setLoginState } = loginActions;
 const signupUrl = 'https://learnground-api-staging.herokuapp.com/api/v1/users';
```

Comment:
Let us not hardcode urls. Make this an env variable.

Reply:
I have made the changes